enterprise/server/workflow/config/config.go

```diff
@@ -167,16 +168,35 @@ func NewConfig(r io.Reader) (*BuildBuddyConfig, error) {
 	return cfg, nil
 }
 
+const kytheDownloadURL = "https://storage.googleapis.com/buildbuddy-tools/archives/kythe-v0.0.67.tar.gz"
+
+func checkoutKythe() string {
+	buf := fmt.Sprintf(`
+export KYTHE_DIR="$BUILDBUDDY_CI_RUNNER_ROOT_DIR/kythe-v0.0.67"
+if [ ! -d "$KYTHE_DIR" ]; then
+  mkdir -p "$KYTHE_DIR"
+  curl -sL "%s" | tar -xz -C "$KYTHE_DIR" --strip-components 1
+fi`, kytheDownloadURL)
+	return buf
+}
+
+func buildWithKythe(cacheURL string) string {
```

Comment:
nit: to avoid this plumbing you could also use `--config=buildbuddy_remote_cache` which gets configured in our generated bazelrc

Reply:
Done.